app/src/components/change_password.rs

```diff
@@ -11,6 +11,8 @@ use crate::{
 use anyhow::{Result, anyhow, bail};
 use gloo_console::error;
 use lldap_auth::*;
+use crate::infra::api;
```

Comment:
Is that how the formatter organizes the imports? Or do you maybe need to run `cargo fmt --workspace`?